search-app/src/cmr/search/results_handlers/reference_results_handler.clj

```diff
@@ -65,14 +66,33 @@
    "revision-id"
    "_score"])
 
+(doseq [concept-type-key (concepts/get-generic-concept-types-array)]
+  (defmethod elastic-search-index/concept-type+result-format->fields [concept-type-key :xml]
+  [concept-type query]
+  ["name"
+   "id"
+   "provider-id"
+   "concept-id"
+   "deleted"
+   "revision-id"
+   "_score"]))
+
+;; TODO: Generic work: Should use a configuration file here?  Or is this OK to have in code?  
+;; Then we need a how to create a new concept for search wiki page or read me or something.
```

Comment:
Please add to a sub-task or checklist item on the parent ticket and remove the TODO

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8621